hitl/run.sh

```diff
@@ -1,36 +1,38 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Title:
 #   Booster hardware-in-the-loop (HITL) test script.
 
 # Enable shell operating mode flags.
 set -eux
 
+BOOSTER="${BOOSTER:-booster-hitl}"
+MQTT_BROKER="${MQTT_BROKER:-mqtt.ber.quartiq.de}"
+# When only one booster is connected, we can use the discovery prefix for all activities.
+PREFIX="${PREFIX:-dt/sinara/booster/+}"
+
 function finish {
     # Always disable the mains supply to Booster using the MQTT-enabled power switch upon
     # completion. Leaving booster enabled can be annoying because of the cooling fans.
-    mosquitto_pub -h mqtt.ber.quartiq.de -t cmnd/tasmota_4F64D0/POWER -m OFF
+    mosquitto_pub -h "${MQTT_BROKER}" -t cmnd/tasmota_4F64D0/POWER -m OFF
 }
 trap finish EXIT
 
-# When only one booster is connected, we can use the discovery prefix for all activities.
-PREFIX=dt/sinara/booster/+
-
 # Set up python for testing
-python3 -m venv --system-site-packages vpy
+test -d vpy || python3 -m venv --system-site-packages vpy
 . vpy/bin/activate
 
 # Install Miniconf utilities for configuring stabilizer.
 python3 -m pip install --upgrade pip
 python3 -m pip install -e py
 
 # Enable the mains supply to Booster using the MQTT-enabled power switch.
-mosquitto_pub -h mqtt.ber.quartiq.de -t cmnd/tasmota_4F64D0/POWER -m ON
+mosquitto_pub -h "${MQTT_BROKER}" -t cmnd/tasmota_4F64D0/POWER -m ON
 
 # Give booster a moment to power up after enabling the mains supply
 sleep 5
 
-probe-rs download --chip STM32F407ZGTx --log-file /dev/null --probe 0483:3754:003C002F5632500A20313236 target/thumbv7em-none-eabihf/release/booster 
```

Comment:
If you feel the need to make it all configurable through environment variables, why not this one?

Reply:
I couldn't test this because I have to flash it manually via DFU since I have no debug probe, so had to comment out the `probe-rs` lines to text locally.

Reply:
If you want the other env variables in, then please add one for the probe as well. to make it useful.